secedgar/client/network_client.py

```diff
@@ -33,11 +35,12 @@ class NetworkClient:
 
     _BASE = "http://www.sec.gov/"
 
-    def __init__(self, retry_count=3, batch_size=10, backoff_factor=0, rate_limit=10):
+    def __init__(self, retry_count=3, batch_size=10, backoff_factor=0, rate_limit=10, user_agent="github.com/sec-edgar/sec-edgar"):
```

Comment:
Forgot to send the review. See it above @kevinschaul

Reply:
I see. Sure, I can adjust it to that if you like. Are there other headers that a user of `sec-edgar` might want to control? Changing the argument from `user_agent` to `headers` will make setting the user agent string a bit more cumbersome, but of course it would let other headers be customized too. I'm happy to make this change if you prefer.

Reply:
> I see. Sure, I can adjust it to that if you like. Are there other headers that a user of `sec-edgar` might want to control? Changing the argument from `user_agent` to `headers` will make setting the user agent string a bit more cumbersome, but of course it would let other headers be customized too. I'm happy to make this change if you prefer.

The code you've written is already good. The only reason I am wondering if providing full access to headers is better is for future changes. Maybe we should just try it out this way until we find (or maybe we won't) a reason to allow users to change all headers. Do you have any thoughts on this @reteps?

Reply:
I think we leave the `user_agent` for now, but if someone requests more headers in the future we can modify it then. I can't think of any headers someone would want. Also, don't want to drag this out, this should be merged ASAP.